web/locales/it.json

```diff
@@ -36,7 +36,7 @@
     },
     "country-history": {
         "carbonintensity24h": "Intensità di carbonio nelle ultime 24 ore",
-        "downloaddataset": "scarica dati completi"
+        "downloaddataset": "Scarica dati completi"
```

Comment:
We've kept it in lower case because there's an icon in front. Can you leave it like that?
I'll make sure to keep all translations in sync if we decide to put uppercase instead